docs/deployments/api-configuration.md

```diff
@@ -44,7 +45,7 @@ Reference the section below which corresponds to your Predictor type: [Python](#
     max_unavailable: <string | int>  # maximum number of replicas that can be unavailable during an update; can be an absolute number, e.g. 5, or a percentage of desired replicas, e.g. 10% (default: 25%)
 ```
 
-See additional documentation for [autoscaling](autoscaling.md), [compute](compute.md), and [prediction monitoring](prediction-monitoring.md).
+See additional documentation for [autoscaling](autoscaling.md), [compute](compute.md), and [prediction monitoring](prediction-monitoring.md) and [system packages](system-packages.md).
```

Comment:
My thinking is this: since 2 new fields can appear in an API, `image` and `tf_serve_image`, you (the user) might want to have a look at how custom images can be built so that they can override the default ones with theirs. What do you think?

Reply:
Ah I see, yes that makes sense. Perhaps we can make it more clear, and replace the hyperlink text with "custom images" or "overriding API images" or similar, what do you think?

Reply:
Yes, I think that would be better. Name them like you said "overriding API images". I'm gonna do it now.